definitions/variable/industry/consumption.yaml

```diff
@@ -0,0 +1,16 @@
+# material demand by commodity and demand sector
+- Consumption|Iron and Steel|Steel|{Consumption Sector}:
+    description: consumption of steel commodities in {Consumption Sector}
```

Comment:
Sorry, I think my previous comment led to a misunderstanding - please capitalize in the description a variable but start with lower-case in the description of tag-lists